### PR TITLE
PXB-2409 Update the CHANGE MASTER TO command to CHANGE REPLICATION SO…

### PR DESCRIPTION
**Pull request by patrickbirch**
…URCE TO (8.0)

 Changes to be committed:
	modified:   howtos/recipes_ibkx_gtid.rst
	modified:   howtos/setting_up_replication.rst
	modified:   xtrabackup_bin/working_with_binary_logs.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst [hunk 1]
```diff
@@ -176,9 +176,9 @@ On the ``Replica``, review the content of the file :file:`xtrabackup_binlog_info
     $ cat /var/lib/mysql/xtrabackup_binlog_info
    Source-bin.000001     481
 
-On the ``Replica``, execute the ``CHANGE MASTER`` statement on a MySQL console and use the username and password you've set up in STEP 3: 
+If you are using version 8.0.22 or earlier, on the ``Replica``, execute the ``CHANGE MASTER`` statement on a MySQL console and use the username and password you've set up in STEP 3:
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst]: would be nice if we can mention version earlier to 8.0.23 as mentioned in other places.

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst]: updated.

File: storage/innobase/xtrabackup/doc/source/howtos/recipes_ibkx_gtid.rst [hunk 2]
```diff
@@ -88,6 +88,26 @@ source node and, finally, start the replica. Run the following commands on the r
...
+    > SET SESSION wsrep_on = 0;
+    > RESET MASTER;
+    > SET SESSION wsrep_on = 1;
+    > SET GLOBAL gtid_purged='<gtid_string_found_in_xtrabackup_binlog_info>';
+    > CHANGE REPLICATION SOURCE TO 
+                SOURCE_HOST="$masterip", 
+                SOURCE_USER="repl",
+                SOURCE_PASSWORD="$slavepass",
+                SOURCE_AUTO_POSITION = 1;
+    > START SLAVE;
```
Comment (george-lorch) [inline storage/innobase/xtrabackup/doc/source/howtos/recipes_ibkx_gtid.rst]: So Oracle change "CHANGE MASTER TO" but did not change "RESET MASTER" to "RESET SOURCE" and "START SLAVE" to "START REPLICA" or something like that?

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/howtos/recipes_ibkx_gtid.rst]: Nope - RESET MASTER has not changed (https://dev.mysql.com/doc/refman/8.0/en/reset-master.html)
I'll update the START SLAVE.

**Review by george-lorch — APPROVED** [review]

**Pull request merged**